eng/helix/content/installaspnetref.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Can we migrate these to C#

Reply:
Yeah i'll file an issue for this and the other one (shared runtime)

Reply:
https://github.com/dotnet/aspnetcore/issues/20624